pkgs/applications/misc/alacritty/default.nix

```diff
@@ -92,6 +92,10 @@ in buildRustPackage rec {
     patchelf --set-rpath "${stdenv.lib.makeLibraryPath rpathLibs}" $out/bin/alacritty
   '') + ''
 
+    install -D alacritty-completions.zsh "$out/share/zsh/site-functions/_alacritty"
+    install -Dt "$out/etc/bash_completion.d" alacritty-completions.bash
+    install -D alacritty-completions.fish "$out/share/fish/vendor_completions.d/alacritty.fish"
```

Comment:
It really bothers me how the second line has the target as the first parameter while the others do not!

Reply:
I could change that, but then it would bother me that `alacritty-completions.bash` turns up twice in that line :D